### ISSUE TITLE
Collect user behavior metrics

### ISSUE THREAD

**Issue by MiklerGM** [chronhq/frontend#20]
New interface introduced in #18 (#7 #8) is not covered with user activity metrics.

Goal | Trigger | Interface element | ID | Finished
-- | -- | -- | -- | --
Open main page | Open main page | header | main_open | <ul><li>- [X] </li></ul> 
Open "about us" page | click on "about us" button | button | check_about | <ul><li>- [X] </li></ul> 
Open license page | click on github link | link | goto_github | <ul><li>- [X] </li></ul> 
Use timepanel | click on timepanel | timepanel | check_timepanel | <ul><li>- [X] </li></ul> 
Use search | search form sent | search | main_search | <ul><li>- [X] </li></ul> 
Open narrative | click on "narrative" button | button | narrative_open | <ul><li>- [X] </li></ul> 
Read narrative | open expanded narrative frame | article | narrative_expand | <ul><li>- [X] </li></ul> 
Complete narrative | click on last narration | dashboard | narrative_completed | <ul><li>- [X] </li></ul> 
Switch on/off cities | click on icon | button | main_cities | <ul><li>- [X] </li></ul> 
Switch on/off borders | click on icon | button | main_borders | <ul><li>- [X] </li></ul> 
Switch on/off people | click on icon | button | main_people | <ul><li>- [X] </li></ul> 
Switch on/off battles | click on icon | button | main_battle | <ul><li>- [X] </li></ul> 
Switch on/off peace treaties | click on icon | button | main_document | <ul><li>- [X] </li></ul> 
Use map | click on map | mapWrapper | check_map | <ul><li>- [X] </li></ul> 
Drag map | mouse drag | mapWrapper | ~drag_map~ | <ul><li>- [ ] </li></ul> 
Open country info | open country tooltip | balloon | check_country | <ul><li>- [X] </li></ul> 
Open wikidata page | click on wikidata link | link | goto_wikidata | <ul><li>- [X] </li></ul> 
Open battle info | on hover battle | balloon | check_battle | <ul><li>- [X] </li></ul> 
Change year via controls | select year | timepanel controls | year_change | <ul><li>- [X] </li></ul> 
-- | -- | -- | -- | --

**Comment by faultcore** [comment]
https://docs.google.com/spreadsheets/d/1pVzAiFzHvBnjbl719bVBMdt3XDC-K0etwMXJHNU_1xA/edit?usp=sharing
Here all the analytics so far

**Comment by faultcore** [comment]
Google ad number UA-133445777-1
```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-133445777-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-133445777-1');
</script>
```
Yandex ad number
```
<!-- Yandex.Metrika counter -->
<script type="text/javascript" >
   (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
   m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)})
   (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");

   ym(52148236, "init", {
        id:52148236,
        clickmap:true,
        trackLinks:true,
        accurateTrackBounce:true,
        webvisor:true
   });
</script>
<noscript><div><img src="https://mc.yandex.ru/watch/52148236" style="position:absolute; left:-9999px;" alt="" /></div></noscript>
<!-- /Yandex.Metrika counter -->
```

**Comment by MiklerGM** [comment]
Fixed in #34
